app/js/dataModels/DiseaseModel.js

```diff
@@ -1,161 +1,138 @@
-function factoryWrapper($log, _, $q, $timeout, $http, AppSettings){
-  'ngInject';
+function factoryWrapper($log, _, $q, $timeout, $http, AppSettings) {
+  "ngInject";
 
-  $log = $log.getInstance('DiseaseModel', false);
-  $log.log('');
+  $log = $log.getInstance("DiseaseModel", false);
+  $log.log("");
 
   const API_BASE = `${AppSettings.api.baseUrl}`;
-  
 
-   /**
+  /**
    * Model constructor and initialization
    * @api public
-   * @param { Object } diseaseResponse  -  disease result object from server 
+   * @param { Object } diseaseResponse  -  disease result object from server
    * @param { Array } mutationsGenes  -   array of gene entrezid, queryBuilder user selected genes
    */
-  function DiseaseModel(diseaseResponse, mutationsGenes){
-    this.samples   = 0;
+  function DiseaseModel(diseaseResponse, mutationsGenes) {
+    this.samples = 0;
     this.negatives = 0;
     this.positives = 0;
     return this.build(diseaseResponse, mutationsGenes);
   }
 
-
-
-   /**
+  /**
    * Populate model with aggregate data from api
    * @api public
-   * @param { Object } diseaseResponse  -  disease result object from server 
+   * @param { Object } diseaseResponse  -  disease result object from server
    * @param { Array } mutationsGenes  -   array of gene entrezid, queryBuilder user selected genes
-   * 
+   *
    * @return Promise - resolved when all aggregate data has been received and processed
```

Comment:
This is also no longer true. We now return a "stub" of a `DiseaseModel` and leave it to the caller to follow up with a call to `.getAggregates()` if and when they choose.